lib/MusicBrainz/Server/Edit/Relationship/Delete.pm

```diff
@@ -248,6 +250,22 @@ sub accept {
         $self->data->{relationship}{id}
     ) or return;
 
+    $self->c->model('Link')->load($relationship);
+    $self->c->model('LinkType')->load($relationship->link);
+    $self->c->model('LinkType')->load_documentation($relationship->link->type);
+    my @examples = $relationship->link->type->all_examples;
+
+    if (any { $_->{relationship}->id == $relationship->id } @examples) {
+        my $error = localized_note(
+            N_l('This edit would remove a relationship that is set '.
+            'as an example of its relationship type in the documentation. '.
+            'If you still think this should be removed, please '. 
```

Comment:
Just coding style: please vertically align strings and put a space before the concatenation operator `.`